### PR TITLE
fix(integ-runner): fails with "no such file or directory, open 'node_modules/@aws-cdk/integ-runner/lib/workers/db.json.gz'"

### PR DESCRIPTION
**Pull request by mrgrain**
Mark the `@aws-cdk/aws-service-spec` as external, because it cannot be bundled.

Changes an import from `cdk-assets` to be more specific, so we don't bundle the whole of `aws-sdk` anymore.
This reduced the worker entrypoint size from 70MB -> 1.5MB

----

*By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license*

### CODE REVIEW

File: packages/@aws-cdk/integ-runner/lib/runner/private/cloud-assembly.ts [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import * as path from 'path';
 import { AssemblyManifest, Manifest, ArtifactType, AwsCloudFormationStackProperties, ArtifactManifest, MetadataEntry, AssetManifestProperties, ArtifactMetadataEntryType, ContainerImageAssetMetadataEntry, FileAssetMetadataEntry } from '@aws-cdk/cloud-assembly-schema';
-import { AssetManifest, FileManifestEntry, DockerImageManifestEntry } from 'cdk-assets';
+import { AssetManifest, FileManifestEntry, DockerImageManifestEntry } from 'cdk-assets/lib/asset-manifest';
```
Comment (kaizencc) [inline packages/@aws-cdk/integ-runner/lib/runner/private/cloud-assembly.ts:3]: nice :)

**Review by kaizencc — APPROVED** [review]

File: packages/@aws-cdk/integ-runner/lib/runner/private/cloud-assembly.ts (hunk 1 above)
Comment (mrgrain) [reply packages/@aws-cdk/integ-runner/lib/runner/private/cloud-assembly.ts:3]: It's ugly. But since it's a monorepo we can do it!

**Comment by AntonioAngelino** [comment]
@mrgrain The cli-lib-alpha is still broken for the same reason.

`Deployment failed: Error: ENOENT: no such file or directory, open '/home/xxx/yyyy/node_modules/@aws-cdk/cli-lib-alpha/db.json.gz'`

**Comment by mrgrain** [comment]
Thanks! I know why. Looking into it when I'm back at work next week.

**Comment by AntonioAngelino** [comment]
@mrgrain let me know if I can help. This issue is a blocker for us. Thank you!

**Comment by mrgrain** [comment]
I have the fix ready. Currently making sure this won't happen again. Unfortunately due to re:Invent it won't be released until next week.

**Comment by AntonioAngelino** [comment]
Ouch. Which old version of aws cdk is not affected by the bug? 

We can think of using an older version or a nightly build (if it's possible).

**Comment by mrgrain** [comment]
[v2.106.0](https://github.com/aws/aws-cdk/releases/tag/v2.106.0) should be the one that introduced the issue, so anything before that should still work. There isn't anything that's strictly forcing you to keep `aws-cdk-lib` at the same version, but if you can I would.

We don't have nightly builds at the moment. If you want to hack it, [this package](https://www.npmjs.com/package/@aws-cdk/aws-service-spec) contains a `db.json.gz` file that needs to be copied to `./node_modules/@aws-cdk/cli-lib-alpha/db.json.gz`.

**Comment by AntonioAngelino** [comment]
Ok, thank you. Hacking is fine for now since we're just developing the solution, ty.  👍

**Pull request merged**